zebra-state/src/request.rs

```diff
@@ -107,4 +107,10 @@ pub enum Request {
     /// Note: the [`HashOrHeight`] can be constructed from a [`block::Hash`] or
     /// [`block::Height`] using `.into()`.
     Block(HashOrHeight),
+
+    /// Request a UTXO identified by the given Outpoint
+    AwaitUtxo(
+        /// The outpoint identifying with the requested UTXO
```

Comment:
TIOLI: I don't think this doc comment adds any info beyond what's already written in the line above.